__tests__/Port.test.js

```diff
@@ -0,0 +1,39 @@
+const Port = require("../src/Port");
+
+describe("Port", () => {
+  describe("ports and ships", () => {
+    let port;
+    let ship;
+    let titanic;
+    let queenMary;
+
+    beforeEach(() => {
+      port = new Port("name");
+      ship = jest.fn();
+      titanic = jest.fn();
+      queenMary = jest.fn();
+    });
+
+    it("can be instantiated", () => {
+      expect(new Port()).toBeInstanceOf(Object);
+    });
+
+    it("port has a name", () => {
+      expect(port.name).toBeDefined();
+    });
```

Comment:
I would add an expect statement to check that `port.name` has the expected value, e.g.
```
expect(port.name).toBe("name");
```

Reply:
Your amendment was how i originally wrote the code. However, during the 'Isolating with Spies' section of the track, I amended it to use a mock function. I guess i didn't need to do that, as best practice was how it was originally written?

Reply:
You should definitely have this test written as I have. If you are defining a value to be something specific you should make sure that it is that value